app/src/controllers/authenticationController.js

```diff
@@ -37,11 +37,12 @@ export default class AuthenticationController {
   /**
    * The Assertion consumer service.
    *
+   * @param user
    * @param req
    * @param res
    */
-  acs(req: express$Request, res: express$Response) {
-    const maybeUser = extractUserFromSpid(req);
+  acs(user: any, req: express$Request, res: express$Response) {
```

Comment:
since this is yet to be validated it should be called in a way that makes it clear that this is not yet a valid user (eg. `userJson` or `userPayload`)